src/scss/components/_dynamic_list.scss

```diff
@@ -0,0 +1,56 @@
+.dynamic-list {
+  &__title {
+    font-size: 21px;
```

Comment:
I'm guessing `21px` is common in other parts of Florence for headings etc - do we not have styling/classes in places to cover these typographies more broadly?

Reply:
Yes, and no. 21px is on the UX design. It is used for titles around other areas in Florence but often declared like `$magicnumber1 * 3` which I could change this to if preferred?